_glaredb/connections-and-integrations/index.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Connections and integrations
```

Comment:
I might make this just "Integrations", since it will clean up the sidebar a bit:

![Sidebar](https://github.com/GlareDB/glaredb.github.io/assets/11184711/7b9e7b0d-caff-4979-9797-e48a4b5c654d)